### ISSUE TITLE
export data to google calendar?

### ISSUE THREAD

**Issue by toobulkeh** [pencilblue/calendar-pencilblue#18]
I'm looking at HQ Raleigh as a production site backed by PencilBlue. I'd like to access their events calendar via Google Calendar.

Investigating this, Google Calendar supports [iCal as a format](https://support.google.com/calendar/answer/37118?hl=en&rd=2). Looks like you use FullCalendar.js, which doesn't export to iCal, but [supports Google Calendar as a backend to it](http://fullcalendar.io/docs/google_calendar/).

I'm not familiar with the PB backend, but it doesn't appear to use Google Calendar as a backend, is that correct? If so, what do you think about exporting to `iCal`? Thanks!

**Comment by blakecallens** [comment]
@toobulkeh the default template for the calendar plugin exports to both Google and iCal [in the event listings](https://github.com/pencilblue/calendar-pencilblue/blob/master/templates/elements/event/event_buttons.html). I checked the HQ site and it those individual event exports are there.

If you're looking to export the entire calendar, you could probably write a controller that uses the same functionality as the individual events, but just loops through them and returns an .ics file.

**Comment by toobulkeh** [comment]
Yup, I'm thinking of a hosted ics file, instead of a one time export
